feature/agenda/presentation/src/main/java/com/sergius/agenda/presentation/task/TaskDetailsViewModel.kt

```diff
@@ -1,6 +1,35 @@
 package com.sergius.agenda.presentation.task
 
+import androidx.compose.material3.ExperimentalMaterial3Api
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.onStart
+import kotlinx.coroutines.flow.stateIn
 
+@OptIn(ExperimentalMaterial3Api::class)
 class TaskDetailsViewModel: ViewModel() {
+
+    private val _state = MutableStateFlow(TaskDetailsState())
+    val state = _state
+        .onStart {  }
+        .stateIn(
+            scope = viewModelScope,
+            started = kotlinx.coroutines.flow.SharingStarted.WhileSubscribed(2000),
```

Comment:
Is there a reason you deviated from the typical official 5s timeout for `WhileSubscribed`? That 5s value comes from Android's ANR timeout which is exactly 5 seconds.